script/run/run.sh

```diff
@@ -34,10 +34,22 @@ fi
 if [ "$(pwd)" != '/' ]; then
     VOLUMES="-v $(pwd):$(pwd)"
 fi
+ARGS="$@"
+while (( $# > 0 )); do
+    if [[ "$1" =~ ^-f || "$1" =~ ^--file ]]; then
+        if [[ "$1" =~ = ]]; then
+            COMPOSE_FILE="${1#*=}"
+        else
+            COMPOSE_FILE="$2"
+        fi
+        break
+    fi
+    shift
+done
+set -- "$ARGS"
```

Comment:
Why ` =~ ^-f` instead of ` == "-f*"` ?

It would be great if this iteration could be done without having to shift off all the items.